app/templates/partials/questions/daterange.html

```diff
@@ -1 +1,42 @@
-{% include theme('partials/questions/general.html') %}
+{% if not question_answers %}
+  {% set question_answers %}
+    <div class="question__answers">
+      {%- for answer in question.answers -%}
+        {% if not answer.parent_answer_id %}
```

Comment:
`parent_answer_id` was removed from `general.html` in https://github.com/ONSdigital/eq-survey-runner/pull/1920. See https://github.com/ONSdigital/eq-survey-runner/blob/master/app/templates/partials/questions/general.html for the new version.

Reply:
I've updated this to match general.html from #1920 now